index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body class="container">
+    <header class="header">
+        <p class="name">Fernando Herrera</p>
+        <div>
+            <div class="hamb hamb1"></div>
+            <div class="hamb hamb2"></div>
+            <div class="hamb hamb3"></div>            
+        </div>     
+    </header>
+    <main >
+        <div class="text-container">
+            <p class="welcome">Hey There. I'm Fernando</p>
```

Comment:
- [ ] Hi, one final thing. This text should use a `h1` tag and not a `p` tag.
something like this would make more sense.
```suggestion
            <h1 class="welcome">Hey There. I'm Fernando</h1>
```